8Knot/pages/contributions/visualizations/pr_first_response.py

```diff
@@ -0,0 +1,225 @@
+from dash import html, dcc, callback
+import dash
+import dash_bootstrap_components as dbc
+from dash.dependencies import Input, Output, State
+import plotly.graph_objects as go
+import pandas as pd
+import logging
+from dateutil.relativedelta import *  # type: ignore
+import plotly.express as px
+from pages.utils.graph_utils import get_graph_time_values, color_seq
+from queries.pr_response_query import pr_response_query as prr
+import io
+from cache_manager.cache_manager import CacheManager as cm
+from pages.utils.job_utils import nodata_graph
+import time
+
+PAGE = "contributions"
+VIZ_ID = "pr-first-response"
+
+gc_pr_first_response = dbc.Card(
+    [
+        dbc.CardBody(
+            [
+                html.H3(
+                    "Pull Request Time to First Response",
+                    className="card-title",
+                    style={"textAlign": "center"},
+                ),
+                dbc.Popover(
+                    [
+                        dbc.PopoverHeader("Graph Info:"),
+                        dbc.PopoverBody(
+                            """
+                            Visualizes number of pull requests that are open over time compared to\n
+                            the number who have gotten a response in the user inputted amoutn of days.
```

Comment:
@JamesKunstle 

On your interpretation: 
- The line plot is created by getting the number of open prs every day, and how many of those open prs have a response within x amount of days. Not so much a "given timeframe" but a plot over time where each day is a point. 
- "this can help the user..." I know before you mentioned keeping this type of information outside of the graph info. Not against giving insight on how to interpret the graph, but would need to be consistent about it so I think this discussion is its own issue if its something youve changed your mind on

Other comments: Definitely like the extension ideas. Definitely open issues for them while they are still fresh in mind. Visualization backlog is getting light so having more thought out concepts will be great

Reply:
re: your first point - okay, so for day 'n', count the num of open PRs total, and the number for which the first response was within the range that the user specifies. That makes sense.

re: your second point - that's true, I did say that, and still believe it. I wasn't super picky when I wrote my suggestion, my bad.

re: other comments - okay will do!

Reply:
@JamesKunstle Any edit that you see for the graph description?

Reply:
One idea that comes to mind:

"Compares the volume of PRs being opened against the number of those PRs that receive at least one response within the parameterized timeframe.

_params:_
* **Response Days** -- Length of timeframe for first response, in days."

I've been thinking about adding a parameters section to graph descriptions, this is one way to do it.

Reply:
@JamesKunstle I like the description. I opened a separate issue for the parameter portion to discuss how that should be implemented but feel like this PR should go forward without the params and then implemented on all graphs in a separate PR. I could see an argument for a different button for it as the graph info is already really dense for some graphs

Reply:
https://github.com/oss-aspen/8Knot/issues/512